### DISCUSSION TITLE
[v13] generateMetadata not having any props, am i missing any configuration?

### DISCUSSION THREAD

**Discussion by VichetN** [vercel/next.js#45890] [Help] [answered]
![image](https://user-images.githubusercontent.com/86401710/218667416-4b1fa4d8-8e54-46f4-82ec-62f797e9ee4e.png)

**Comment by icyJoseph** [comment]
Just a quick check, what version of next.js are you using?

**Reply by VichetN** [reply]
It is Next v13.1.6.

**Reply by eagor** [reply]
I'm experiencing the same issue. 
ver 13.1.6

**Comment by eagor** [comment, marked as answer]
[13.1.7-canary.12](https://github.com/vercel/next.js/releases/tag/v13.1.7-canary.12) seems to fix the issue

**Reply by VichetN** [reply]
Whoo Hoo, Work like magic, Thanks @eagor

**Reply by jee-r** [reply]
I'm also concerned by this issue. 
Does anyone know what the requirements are for the pre-release (canary release) to become the stable (latest) release?
